director-certs.html.md.erb

```diff
@@ -8,16 +8,27 @@ Depending on you configuration, there are up to three endpoints to be secured us
 
 <p class="note">Note: If you are using the UAA for user management, an SSL certificate is mandatory for the Director and the UAA.</p>
 
-You can use the following script to generate a root CA certificate and use it to sign three generated SSL certificates:
+You can use the following script `director_certs.sh` to generate a root CA certificate and use it to sign three generated SSL certificates:
 
 <pre class="bash">
 #!/bin/bash
 
 set -e
 
-certs=`dirname $0`/certs
+ip=$1
+if [[ "${ip}X" == "X" ]]; then
+  echo "USAGE: ./director_certs.sh [public IP | path to bosh-init manifest]"
+  exit 1
+fi
+if [[ -f ${ip} ]]; then
+  ip=$(cat ${ip} | yaml2json | jq -r ".jobs[] | select(.templates[].name | contains(\"director\")).networks[0].static_ips[0]")
+fi
```

Comment:
Can we just have the IP as user input instead of doing this cat-yaml2json-jq-spruce stuff?